ci/bin/build-and-test

```diff
@@ -7,7 +7,19 @@ template_path = File.join(root_path, "template.rb")
 builds_path = File.join(root_path, "tmp/builds")
 app_name = ENV.fetch("APP_NAME", "app")
 app_path = File.join(builds_path, app_name)
-config_path = File.join(root_path, ENV.fetch("CONFIG_PATH", "ci/configs/basic.yml"))
+
+config_path = if !ENV.key?("CONFIG_PATH") || ENV["CONFIG_PATH"] == ""
```

Comment:
can't we do

```suggestion
config_path = if ENV.fetch("CONFIG_PATH", "").blank?
```

Reply:
@G-Rath `#blank?` comes from ActiveSupport, which is not loaded here. I will adjust your suggestion though to use `fetch`